plugins/main/bridge/bridge.go

```diff
@@ -75,6 +75,9 @@ func loadNetConf(bytes []byte) (*NetConf, string, error) {
 	if err := json.Unmarshal(bytes, n); err != nil {
 		return nil, "", fmt.Errorf("failed to load netconf: %v", err)
 	}
+	if n.Vlan < 0 || n.Vlan > 4094 {
+		return nil, "", fmt.Errorf(`invalid VLAN ID %d (must be between 0 and 4094)`, n.Vlan)
```

Comment:
Why there you are using backtick instead of commonly used `"`?

Reply:
@mars1024 we all agree that if you change to " then we'll merge :) Thanks!

Reply:
@jellonek @dcbw definitely OK, btw, I will change the [usage](https://github.com/containernetworking/plugins/blob/15894b36a0a679028768bf02498618992e808136/plugins/main/vlan/vlan.go#L59) in vlan plugin too.

Reply:
Updated, thanks!